src/lib/components/editor/AiTranslateToolbarButton.svelte

```diff
@@ -100,8 +111,23 @@
         editor.commands.setContent(fullContent);
     };
 
-    const onClick = async () => {
-        const originalHtml = generateHTML(editor.getJSON(), extensions(false, undefined, false, undefined));
+    const onClick = async (retranslating = false) => {
```

Comment:
Could we rename this function now that it's being called from inside other code in the file?